<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/imgP/Recursos/dinero.jpg</normaloff>:/imgP/Recursos/dinero.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 209, 176, 255), stop:1 rgba(251, 255, 215, 255));
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nombre: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="inNombre">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(253, 255, 237);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Ingrese el nombre del obrero</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Horas:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="inHoras">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(253, 255, 237);</string>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Jornada</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QRadioButton" name="inMatutina">
               <property name="text">
                <string>Matutina </string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="inVespertina">
               <property name="text">
                <string>Vespertina</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="inNocturna">
               <property name="text">
                <string>Nocturna</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>112</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="cmdCalcular">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(253, 255, 237);</string>
               </property>
               <property name="text">
                <string>&amp;Calcular</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="outResultado">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(253, 255, 237);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo </string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionCalcular"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imgP/Recursos/new.png</normaloff>:/imgP/Recursos/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imgP/Recursos/open-box.png</normaloff>:/imgP/Recursos/open-box.png</iconset>
   </property>
   <property name="text">
    <string>A&amp;brir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imgP/Recursos/guardar.png</normaloff>:/imgP/Recursos/guardar.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imgP/Recursos/exit.jpg</normaloff>:/imgP/Recursos/exit.jpg</iconset>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>A&amp;cerca de</string>
   </property>
  </action>
  <action name="actionCalcular">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imgP/Recursos/calculator (1).png</normaloff>:/imgP/Recursos/calculator (1).png</iconset>
   </property>
   <property name="text">
    <string>Calcular</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
